.github/workflows/docusaurus.yml

```
@@ -0,0 +1,56 @@
+name: Deploy website
+
+# This workflow is triggered on a pull request. This is a single event so no brackets are needed.
+on: pull_request
+  paths: 
+  - site/docs/
```

I didn't know it was that easy! Nice job!

```suggestion
  - site/docs/
  - site/website/
```

There are some files in `site/website` that should also trigger a redeployment of the site.